plugins/org.jboss.tools.openshift.common.core/src/org/jboss/tools/openshift/internal/common/core/util/CommandLocationLookupStrategy.java

```diff
@@ -29,12 +29,22 @@
 	private static final String LINUX_PATHVAR = "PATH";
 	private static final String WINDOWS_PATHVAR = "Path";
 
+	private static final String LINUX_SEPARATOR = ":";
```

Comment:
After adding static import

``` java
import static java.io.File.pathSeparator;
```

This two vars can be removed and replaced to simple 'pathSeparator'.